hw/xbox/nv2a/nv2a_debug.h

```diff
@@ -41,25 +42,26 @@ void gl_debug_group_end(void);
 void gl_debug_label(GLenum target, GLuint name, const char *fmt, ...);
 void gl_debug_frame_terminator(void);
 
-# define NV2A_GL_DPRINTF(cc, format, ...) \
-    gl_debug_message(cc, "nv2a: " format, ## __VA_ARGS__)
-# define NV2A_GL_DGROUP_BEGIN(format, ...) \
-    gl_debug_group_begin("nv2a: " format, ## __VA_ARGS__)
-# define NV2A_GL_DGROUP_END() \
-    gl_debug_group_end()
-# define NV2A_GL_DLABEL(target, name, format, ...)  \
-    gl_debug_label(target, name, "nv2a: { " format " }", ## __VA_ARGS__)
-#define NV2A_GL_DFRAME_TERMINATOR() \
-    gl_debug_frame_terminator()
+#define NV2A_GL_DPRINTF(cc, format, ...) \
+    gl_debug_message(cc, "nv2a: " format, ##__VA_ARGS__)
+#define NV2A_GL_DGROUP_BEGIN(format, ...) \
+    gl_debug_group_begin("nv2a: " format, ##__VA_ARGS__)
+#define NV2A_GL_DGROUP_END() gl_debug_group_end()
+#define NV2A_GL_DLABEL(target, name, format, ...) \
+    gl_debug_label(target, name, "nv2a: { " format " }", ##__VA_ARGS__)
+#define NV2A_GL_DFRAME_TERMINATOR() gl_debug_frame_terminator()
 
 #else
-# define NV2A_GL_DPRINTF(cc, format, ...)          do { \
-        if (cc) NV2A_DPRINTF(format "\n", ##__VA_ARGS__ ); \
+#define NV2A_GL_DPRINTF(cc, format, ...)              \
+    do {                                              \
+        if (cc) {                                     \
+            NV2A_DPRINTF(format "\n", ##__VA_ARGS__); \
+        }                                             \
     } while (0)
-# define NV2A_GL_DGROUP_BEGIN(format, ...)         do { } while (0)
-# define NV2A_GL_DGROUP_END()                      do { } while (0)
-# define NV2A_GL_DLABEL(target, name, format, ...) do { } while (0)
-# define NV2A_GL_DFRAME_TERMINATOR()               do { } while (0)
+#define NV2A_GL_DGROUP_BEGIN(format, ...)         do { } while (0)
+#define NV2A_GL_DGROUP_END()                      do { } while (0)
+#define NV2A_GL_DLABEL(target, name, format, ...) do { } while (0)
+#define NV2A_GL_DFRAME_TERMINATOR()               do { } while (0)
```

Comment:
These we indented to make it easier to see the `#if` / `#else` block.

Reply:
Indifferent here, clang-format auto cleaned these

Reply:
I think we should either indent them, or at least have an empty line around the `#if` `#else` `#endif` so it's more obvious what's going on.

Reply:
I'll go w/ blank line.